tracing/README.md

```diff
@@ -93,44 +89,81 @@ be invoked with the same syntax as the similarly-named macros from the `log`
 crate. Often, the process of converting a project to use `tracing` can begin
 with a simple drop-in replacement.
 
-Let's consider the `log` crate's yak-shaving example:
+Let's consider the `log` crate's yak-shaving
+[example](https://docs.rs/log/0.4.6/log/index.html#examples), tweeked to
+support tracing:
 
 ```rust
-#[macro_use]
-extern crate tracing;
-use tracing::field;
+// Add a use statement so we get tracing version of macros
+use tracing::{span, info, warn, Level};
+
+// Dummy impls to make the example compile
```

Comment:
Perhaps these should be moved to the bottom of the example, after the important parts?